packages/website/ts/components/portal/portal.tsx

```diff
@@ -241,6 +269,140 @@ export class Portal extends React.Component<PortalProps, PortalState> {
             </div>
         );
     }
+    private _renderMenu(routeComponentProps: RouteComponentProps<any>): React.ReactNode {
+        return (
+            <div>
+                <BackButton to={`${WebsitePaths.Portal}`} labelText={'back to Relayers'} />
+                <PortalMenu selectedPath={routeComponentProps.location.pathname} />
+            </div>
+        );
+    }
+    private _renderWallet(): React.ReactNode {
+        const allTokens = _.values(this.props.tokenByAddress);
+        const trackedTokens = _.filter(allTokens, t => t.isTracked);
+        return (
+            <div>
+                <Title labelText={'Your Account'} />
+                <Wallet
+                    userAddress={this.props.userAddress}
+                    networkId={this.props.networkId}
+                    blockchain={this._blockchain}
+                    blockchainIsLoaded={this.props.blockchainIsLoaded}
+                    blockchainErr={this.props.blockchainErr}
+                    dispatcher={this.props.dispatcher}
+                    tokenByAddress={this.props.tokenByAddress}
+                    trackedTokens={trackedTokens}
+                    userEtherBalanceInWei={this.props.userEtherBalanceInWei}
+                    lastForceTokenStateRefetch={this.props.lastForceTokenStateRefetch}
+                    injectedProviderName={this.props.injectedProviderName}
+                    providerType={this.props.providerType}
+                    onToggleLedgerDialog={this._onToggleLedgerDialog.bind(this)}
+                    onAddToken={this._onAddToken.bind(this)}
+                    onRemoveToken={this._onRemoveToken.bind(this)}
+                />
+            </div>
+        );
+    }
+    private _renderAccountManagement(): React.ReactNode {
+        return this.props.blockchainIsLoaded ? (
+            <Switch>
+                <Route path={`${WebsitePaths.Portal}/weth`} render={this._renderEthWrapper.bind(this)} />
+                <Route path={`${WebsitePaths.Portal}/account`} render={this._renderTokenBalances.bind(this)} />
+                <Route path={`${WebsitePaths.Portal}/trades`} render={this._renderTradeHistory.bind(this)} />
+                <Route path={`${WebsitePaths.Portal}/direct`} render={this._renderTradeDirect.bind(this)} />
+                <Route render={this._renderNotFoundMessage.bind(this)} />
+            </Switch>
+        ) : (
+            <div className="pt4 sm-px2 sm-pt2 sm-m1" style={{ height: 500 }}>
```

Comment:
put a TODO here for consolidating this loading component with the one in relayer_index